docs/zk-stack/running-a-hyperchain/locally.md

```diff
@@ -27,20 +33,130 @@ zk
+     - If you are doing this for the first time, several components need to be compiled/built, so do not worry if it takes a few minutes. The console will show what is going on anyways.
 
-::: warning
+   - If you don't want to configure any values for now and just want check the build process for a hyperchain, try out the `zk stack demo` command.
 
-The commands above are not just running docker containers, but are actually building the code from the repo to spin up your hyperchain. For this reason the process might take some time. If you just want to run docker containers to play around with a zkSync chain, you can use zksync-cli dev. [Learn more here](../../build/tooling/zksync-cli/getting-started.md).
+:::warning
 
+The commands above are not just running docker containers, but are actually building the code from the repo to spin up your hyperchain. For this reason the process might take some time. If you just want to run docker containers to play around with a zkSync chain, you can use zksync-cli dev. Learn more here.
 :::
 
 Your hyperchain is now deployed to the base chain (most likely a local geth docker container) and configured. You can find all configuration in a new `.env` file created on `<project root>/etc/env/<your_chain_name_slug>.env`, and if you deployed test tokens, their addresses will be available at `<project root>/etc/tokens/<the_l1_identifier>.json`
+
+1. The wizard allows you to run the server in the end. If you chose not to, you’re still able to run it by executing
+
+```bash
+zk server --components "http_api,eth,state_keeper,housekeeper"
+```
+
+2. You can now run transactions and start playing with your hyperchain by using the RPC available at<http://localhost:3050>.
+
+   - Don't forget to deposit some ETH and fund your accounts on your hyperchain. To do so follow these instructions ->
+
+## Using your hyperchain
+
+During the `zk stack init` configurator, you have a choice of what base layer to deploy the hyperchain onto: local geth node, or an Ethereum testnet. The first step to start interacting with your hyperchain is to fund an account (or a few). This means you need some funds on the base layer ->
+
+### Base layer is the local geth node (<a href="https://hub.docker.com/r/matterlabs/geth" target="_blank">@matterlabs/geth:latest</a>)
+
+You will have a set of addresses that have 100 ETH each. You can find the list here and use these addresses to deposit into your hyperchain.
+
+### Base layer is an Ethereum network (e.g., Goerli, Sepolia)
+
+In this case, you need to have an account on the base layer with ETH (which, for sure, you have given the deployer, governor, and operator needed ETH to go through the deployment process). You can use any of these wallets (or any other you have funds) to deposit into your hyperchain.
+
+Once you have the accounts with funds on the base layer, you can do a deposit (and any further interactions) in 3 ways ->
```

Comment:
not sure what the "->" means here. If it's just pointing the 3 different ways to use the hyperchain RPC mentioned below, I'd be specific about it

Reply:
Changed back to ":"